### DISCUSSION TITLE
Model ideas

### DISCUSSION THREAD

**Discussion by pfeatherstone** [davisking/dlib#2673] [Ideas]
@arrufat I think you have an interest in self-supervised learning. https://github.com/facebookresearch/VICRegL This could be cool and maybe a cool feature for dlib

**Comment by arrufat** [comment]
Thank you, I'm aware of this paper. I think you already recommended it some time ago.

These days I don't have as much time as I used to, though. And if I remember correctly, its performance was very similar to Barlow Twins.

**Reply by arrufat** [reply]
Oh, it's a different one from VICReg: https://arxiv.org/abs/2105.04906, I got confused.
